### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by ghost**
files, images and folders

### CODE REVIEW

File: img/README.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+for slider
```
Comment (jigyasa-grover) [inline img/README.txt:1]: Is this file required ?

File: js/README.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+For slider
```
Comment (jigyasa-grover) [inline js/README.txt:1]: Is this `README.txt` required ?

**Review by jigyasa-grover — COMMENTED** [review]
Provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

File: index.html [hunk 3]
```diff
@@ -47,41 +47,54 @@
...
-            <div class="ism-caption ism-caption-1" data-delay="200">
-                Educating and inspiring<br>
-                students by being involved in<br>
-                open-source projects.<br>
-                <a href="//codein.withgoogle.com/organizations/fossasia/">
-                    Participate now!</a>
-            </div>
-          </li>
-          <li>
-            <img src="images/image_slider_machine.jpg">
```
Comment (niccokunzmann) [inline index.html:74]: Please comment in the commit message, why you have deleted the following sliders.

File: index.html [hunk 4]
```diff
@@ -47,41 +47,54 @@
...
...g=e.slice(d*4,c.min(d*4+4,f)),h=new m(g[0].J,a||0);i(h,g,a);j.Y(h,a)}else b.c(e,function(c){b.c(c.Db,function(b){b.jb(c.Cc||0);j.Y(b,a)})})}a.Gc=function(){a.C(-1,d)};w=[f.Ld,f.Id,f.Hd,f.Fd,f.Ed,f.Kd,f.Sd,f.he,f.ge,f.je,f.be,f.Zd,f.sd,f.Xd,f.Vd,f.Ud,f.Td,f.Qd,f.ke,f.ae,f.Wd,f.Yd,f.ce,f.Rd,f.Cd,f.Od,f.Ke,f.df,f.ff,f.cg,f.bg,f.Zf,f.ef,f.Xf,f.Wf,f.Vf,f.Sf];var C={i:"y",j:"x",D:"m",E:"t",db:"r",V:"rX",Z:"rY",v:"sX",p:"sY",cb:"tX",gb:"tY",W:"tZ",Ub:"kX",Qb:"kY",pb:"o",X:"e",kb:"i",f:"c"};b.c(C,function(b,a){n[b]=a});y(s(v,1));i(e,o);var x=g.Ig||[],k=[].concat(x[b.Lb(b.g(v,"b"))]||[]);!k.length&&k.push({b:e.nb(),d:u});A(k);i(a,h,a.nb());a.C(-1)};jssor_1_slider_init=function(){var h=[[{b:-1,d:1,o:-1},{b:0,d:1e3,o:1}],[{b:1900,d:2e3,x:-379,e:{x:7}}],[{b:1900,d:2e3,x:-379,e:{x:7}}],[{b:-1,d:1,o:-1,r:288,sX:9,sY:9},{b:1e3,d:900,x:-1400,y:-660,o:1,r:-288,sX:-9,sY:-9,e:{r:6}},{b:1900,d:1600,x:-200,o:-1,e:{x:16}}]],j={Xb:d,Eb:800,Ec:f.sd,pg:{G:t,Bb:h},vg:{G:s},ug:{G:r}},e=new i("jssor_1",j);function a(){var b=e.N.parentNode.clientWidth;if(b){b=c.min(b,1920);e.of(b)}else g.setTimeout(a,30)}a();b.a(g,"load",a);b.a(g,"resize",a);b.a(g,"orientationchange",a)}})(window,document,Math,null,true,false)
```
Comment (niccokunzmann) [inline index.html:56]: There is a lot of new JavaScript. It should be in a file.

**Comment by niccokunzmann** [comment]
@nolik2000 please tell me what you did and why (that should be in a commit message).

**Review by daminisatya — CHANGES_REQUESTED** [review]
Requesting changes being addressed by the mentors @niccokunzmann @jig08

**Comment by tradzik** [comment]
Outdated - please see #249 . Closing.
